### PR TITLE
Feature/roc 5350 rpa new judgment types

### PR DESCRIPTION
**Pull request by daveoh**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-5350

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [X] No

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -349,32 +349,35 @@ sonarqube {
...
+        "**/CCDCaseHandler.java",
+        "**/CCDReDetermination.java",
+        "**/CCDEventProducer.java",
+        "**/GeneratedDocuments.java",
+        "**/DBCaseRepository.java",
+        "**/MapperUtil.java",
+        "**/CCDCaseApi.java",
+        "rpa-mapper/**",
+        "**/DataFixSupportController.java"
+      ].join(",")
```
Comment (akritis22) [inline build.gradle:380]: why is this changed?

Comment (daveoh) [reply build.gradle:380]: When merging this a comma was missed out in the string concatenation causing sonar cloud check to fail. It was hard to track down. Changing this to a list will make any issues like this obvious as it wont compile.
